packages/core/src/animation/internal/AnimatorStatePlayData.ts

```diff
@@ -29,7 +29,7 @@ export class AnimatorStatePlayData {
     const duration = state._getDuration();
     this.playState = AnimatorStatePlayState.Playing;
     if (state.wrapMode === WrapMode.Loop) {
-      time = time % duration;
+      time = duration ? time % duration : time;
```

Comment:
you mean duration maybe 0 if only one frame?

Reply:
done